src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -111,13 +112,6 @@ const EditableVariableRow = ({index, variable, onDelete, onChange}) => {
   const prefillPluginChoices = availablePrefillPlugins.map(plugin => [plugin.id, plugin.label]);
   const [prefillAttributeChoices, setPrefillAttributeChoices] = useState([]);
 
-  const formSteps = formContext.formSteps;
-  const formStepsChoices = formSteps.map(step => {
-    if (step.formDefinition) return [step.formDefinition, step.name];
-
-    return [step._generatedId, step.name];
-  });
-
```

Comment:
This seemed not to be used anywhere :shrug:

Reply:
Doesn't look like it had side-effects, so removal is probably ok.